tests/Repository/EmailTemplateTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+ */
+
+declare(strict_types=1);
+
+namespace Tests\BitBag\SyliusMailTemplatePlugin\Repository;
+
+use PHPUnit\Framework\Assert;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+final class EmailTemplateTest extends WebTestCase
+{
+    private $emailTemplateRepository;
```

Comment:
Where is the property type?

Reply:
Fixed

Reply:
Why you're not using PHP7.4 typing? Does the plugin allow the lower PHP version than 7.4?